src/app/components/categories-table/categories-table.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit } from '@angular/core';
+import { Category, CategoryToShow } from 'src/app/models/category';
+import { EndpointsService } from 'src/app/services/endpoints.service';
+import { DomSanitizer } from '@angular/platform-browser';
+import { ConfirmComponent } from '../dialogs/confirm/confirm.component';
+import { Observable } from 'rxjs';
+import { DialogService } from 'src/app/services/dialog.service';
+import { MatTableModule } from '@angular/material/table';
```

Comment:
The modules shouldn't be imported in the component files. 
Remove MatTableModule and MatIconModule from here.